source/CommandLine/ICommandOutputProvider.cs

```diff
@@ -0,0 +1,34 @@
+using System;
+using System.IO;
+using Octopus.CommandLine.OptionParsing;
+using Serilog.Events;
+
+namespace Octopus.CommandLine
+{
+    public interface ICommandOutputProvider
```

Comment:
We've dropped:
```
        void ServiceMessage(string messageName, object o);
        void TfsServiceMessage(string serverBaseUrl, ProjectResource project, ReleaseResource release);
        bool ServiceMessagesEnabled();
        bool IsVSTS();
        void EnableServiceMessages();
```
And pushed them down into an OctopusCLI specific implementation